creator-node/src/serviceRegistry.js

```diff
@@ -44,18 +44,22 @@ class ServiceRegistry {
   /**
    * Configure all services
    */
-  async initServices () {
+  async initServices (initConfig = { blacklistManager: true, libs: true, monitoringQueue: true }) {
```

Comment:
see my comments in the FileProcessingQueue. the problem here is you'd need to explicitly set everything else to false to optimize initializing serviceRegistry